microsoft-edge/index.yml

```diff
@@ -42,33 +42,34 @@ highlightedContent:
 
 # -----------------------------------------------------------------------------
 # Card r1c4
-    - title: "Find out what's new"
-      itemType: learn
-      url: https://www.microsoft.com/microsoft-365/roadmap?filters=Microsoft%20Edge   # or https://www.microsoft.com/edge/update/latest (1 at a time)
+    - title: "Microsoft Edge Roadmap"
+      itemType: reference
```

Comment:
global for Business links: check itemType

Reply:
did global pass at top & bottom of hub page.  3 items have a possibly better type indicated, eg:
```
            - text: Manage Microsoft Edge for Business on iOS and Android with Intune
              url: /mem/intune/apps/manage-microsoft-edge  # https://learn.microsoft.com/mem/intune/apps/manage-microsoft-edge
              itemType: reference  # or how-to-guide
```

Reply:
changed to the alt values that seemed to fit better